_assignments/mp4_nolab.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: assignment
+title: "MP4: Flight Simulator"
+index: 41
+due: Apr. 13, 2020 @ 11:59 PM
+material:
+  -
+    name: MP4 Unity Package
+    url: /mps/MP4.unitypackage
+rubric:
+  -
+    name: Plane
+    points: 5
+    description: Game has a controllable plane
+  -
+    name: Acceleration/Deceleration
+    points: 10
+    description: Plane can accelerate and decelerate comfortably
+  -
+    name: Rotation
+    points: 10
+    description: Plane can yaw, pitch, and roll
+  -
+    name: Weapon
+    points: 10
+    description: Pressing a button shoots out a raycast or bullet from the plane
+  -
+    name: Spheres
+    points: 5
+    description: There are spheres around the terrain
+  -
+    name: Shooting Game
+    points: 25
+    description: Player can shoot spheres, which disappear, then reappear
+  -
+    name: Start Menu
+    points: 10
+    description: Game has a legible and understandable in-world start menu
+  -
+    name: Score Display
+    points: 5
+    description: Score is displayed in-world
+  -
+    name: Sound
+    points: 15
+    description: Sounds in game world complement the environment
+  -
+    name: Game Over
+    points: 15
+    description: Game handles plane crashes and shows game over in some manner
+  -
+    name: Framerate/Comfort
+    points: 40
+    description: Framerate rarely drops below 60 fps, game sound is not obnoxious, and plane acceleration, deceleration, and rotations are comfortable. Game conforms to Oculus Best Practices Guide
+  -
+    name: In-world tutorial
+    points: 20
+    extra: true
+    description: The tutorial and explanation to the player is in game (extra credit)
+
+points: 150
+---
+
+## NO VR
+
+The VR lab is closed, so this will be a regular Unity project with no VR additions.
+
+## Overview
+
+In this MP, you will build a flight simulator in Unity. You must make a start menu, tutorial, script to support your plane, make a UI, and optimize the game to fulfill requirements from the [Oculus Best Practices Guide](https://developer.oculus.com/design/latest/concepts/book-bp/)!
```

Comment:
How are we expecting them to fulfill Oculus Best Practices, I think we should remove this or explain further

Reply:
fixed